src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -58,13 +108,40 @@ class IntroductionPage extends React.Component {
             </li>
           </ol>
         </div>
-        <SaveInProgressIntro
+        {(!signingIn && !sessionStorage.userToken) && <SaveInProgressIntro
```

Comment:
Is there any way to consolidate these `<SaveInProgressIntro>`s?

Reply:
Good idea, updated